labels/en-US.json

```diff
@@ -804,6 +804,7 @@
   "submit": "Submit",
   "submitForApproval": "Submit for Approval",
   "submittedBy": "Submitted By",
+  "subscriptionCancelledBlob1": "Hello!<br/><br/>The status of your subscription #{0} is updated to {1}. If you would like to view or manage your subscription, please visit <a href=\"https://{2}/myaccount\">My Account</a>.",
```

Comment:
Travis, My Account should not be a link, it will be in plain text. I have also made similar mistake then, we've confirmed that with Aalekh and we now we don't need a hyperlink for 'My Account'. So, you can remove it.